code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -816,7 +816,7 @@
 	minimum_players = 25
 	weight = 4
 	cost = 8
-	requirements = list(101,101,101,80,60,50,30,20,10,10)
+	minimum_players = 23
```

Comment:
Sentient disease already has minimum players defined a bit above

Reply:
Good catch, I'm surprised linter doesn't catch this

Reply:
Haha, I only caught it myself because I have placed it there recently